packages/terra-clinical-onset-picker/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "terra-clinical-onset-picker",
+  "version": "0.1.0",
```

Comment:
Start this at version 0.0.0. It will be updated when it gets released.

Reply:
Updated to version 0.0.0